src/Stmt.cc

```diff
@@ -1226,7 +1226,7 @@ ValPtr ForStmt::DoExec(Frame* f, Val* v, StmtFlowType& flow) {
 bool ForStmt::IsPure() const { return e->IsPure() && body->IsPure(); }
 
 void ForStmt::StmtDescribe(ODesc* d) const {
-    Stmt::StmtDescribe(d);
+    Stmt::StmtDescribe(d); // NOLINT(bugprone-parent-virtual-call)
```

Comment:
Looking at `Stmt::StmtDescribe()`, isn't this just running `AddTag(d)` due to `Tag () != STMT_EXPR`. Calling `AddTag(d)` directly here might be more obvious and avoids the eye-brow raising NOLINT :-)

Reply:
Sure. The lint note was because calling `ExprStmt::StmtDescribe` causes the output to differ for `ForStmt` that what we're currently outputting.

For example:

```
for ([HTTP::_], HTTP::c in HTTP::f$conns) { if (HTTP::c?$http) return (HTTP::build_url_http(HTTP::c$http))}return ()}
```

turns into

```
for HTTP::f$conns([HTTP::_], HTTP::c in HTTP::f$conns) { if (HTTP::c?$http) return (HTTP::build_url_http(HTTP::c$http))}return ()}
```